specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/compute.json

```diff
@@ -8383,6 +8412,16 @@
         "managedDisk": {
           "description": "The managed disk parameters.",
           "$ref": "#/definitions/VirtualMachineScaleSetManagedDiskParameters"
+        },
+        "diskIOPSReadWrite": {
+          "type": "integer",
```

Comment:
Is this not 'readOnly' in VMSS?  Can a user set this value only for VMSS and not for VM?